### PR TITLE
Create API readiness checklist for KYC-match

### LINKED ISSUE #107: KYC Match Readiness Checklist
Include the checklist for meta-release Fall24, including documentation, ATPs and alignment with commonalities (0.4.0)

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

Add one of the following kinds:

* documentation
* subproject management

#### Which issue(s) this PR fixes:

Fixes #107 

```

### CODE REVIEW

File: documentation/SupportingDocuments/API-Readiness-Checklist_kyc-match-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# API Readiness Checklist
+
+Checklist for kyc-maych 0.2.0 in FALL24
```
Comment (maxl2287) [inline documentation/SupportingDocuments/API-Readiness-Checklist_kyc-match-0.2.0.md:3]: Typo: ` kyc-maych`

**Review by GillesInnov35 — DISMISSED** [review]
Thanks

File: documentation/SupportingDocuments/API-Readiness-Checklist_kyc-match-0.2.0.md (hunk 1 above)
Comment (jgarciahospital) [reply documentation/SupportingDocuments/API-Readiness-Checklist_kyc-match-0.2.0.md]: solved, thanks!

**Review by GillesInnov35 — DISMISSED** [review]

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

File: code/kyc-match-API-Readiness-Checklist.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# API Readiness Checklist
+
+Checklist for kyc-match 0.2.0 in FALL24
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | link to be provided when release is created, API Code |
```
Comment (hdamker) [inline code/kyc-match-API-Readiness-Checklist.md:7]: You can use here a relative link which will always point to the version within the same release tag or same branch:

```suggestion
|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | [link](/code/API_definitions/kyc-match.yaml) |
```

Comment (hdamker) [reply code/kyc-match-API-Readiness-Checklist.md:7]: Same for the other links.

**Comment by jgarciahospital** [comment]
Already included in the RC branch

**Pull request closed without merging**